README.rst

```diff
@@ -778,6 +787,9 @@ The options (``[flags]``) are as follows:
 * ``--unit-params-file`` (used for ``rpm`` and ``deb`` packages) is is the path to the file that contains
   unit parameters for the ``systemd`` unit files.
 
+**Note**: Please use rockspec name in lower case when you pack
+your application with ``--spec`` flag.
+
```

Comment:
I don't like that this note is far from the description of the flag. See comment above.